app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -31,7 +32,14 @@ export function getStatus(
     approved: publishStatus === NimbusExperimentPublishStatus.APPROVED,
     review: publishStatus === NimbusExperimentPublishStatus.REVIEW,
     waiting: publishStatus === NimbusExperimentPublishStatus.WAITING,
-    endRequested: experiment?.statusNext === NimbusExperimentStatus.COMPLETE,
+    // TODO: EXP-1325 Need to check something else here for end enrollment in particular?
+    pauseRequested:
+      status === NimbusExperimentStatus.LIVE &&
+      statusNext === NimbusExperimentStatus.LIVE &&
+      isEnrollmentPausePending === true,
```

Comment:
The only LIVE>LIVE update we have right now is pausing so that should be sufficient to know we're in a pausing flow?

Reply:
Wanted to be specific here and only catch the pending end enrollment case

Reply:
Oh I see what you mean.  I think we'll end up having to do a lot more infrastructure before we do any non pause live updates, so we shouldn't really need to worry about it for now.